### PR TITLE
Stats: Coding Standards updates

### PR DESCRIPTION
**Pull request by zinigor**
Cleanup modules/stats.php using WordPress php Coding Standards. (Replay of #5398)

### CODE REVIEW

**Comment by zinigor** [comment]
I have added some fixes to this PR, so now I can't approve it. But it should be good to go.

**Comment by singerb** [comment]
Was this intended to fix all standards issue in the file? I'm still seeing quite a bit of super-global access warnings, which are probably fine, but there are a lot of other ones: a few `if`'s without curly braces, missing spaces, missing translator comments, etc. It's certainly better, and if the intention is just to merge this small step then that's fine. Just wanted clarification.

**Comment by zinigor** [comment]
@singerb I think that the intention was to make it better :) And even if it didn't fix all of the stuff - yeah, there's a lot of code that doesn't follow the standards. That's just a step towards the end goal.

**Comment by dereksmart** [comment]
Looks good!

File: modules/stats.php [hunk 1]
```diff
@@ -21,6 +23,12 @@
...
 add_action( 'jetpack_modules_loaded', 'stats_load' );
 
+/**
+ * Load Stats.
+ *
+ * @access public
+ * @return void
+ */
 function stats_load() {
 	global $wp_roles;
```
Comment (eliorivero) [inline modules/stats.php]: This var is unused and should be removed.

File: modules/stats.php [hunk 2]
```diff
@@ -188,30 +245,61 @@ function stats_get_options() {
 	return $options;
 }
 
+/**
+ * Get Stats Options.
+ *
+ * @access public
+ * @param mixed $option Option.
+ * @return Options|null.
```
Comment (eliorivero) [inline modules/stats.php]: There are a bunch of these `Options`. Aren't they `array` instead? We don't have a class named `Options`.

File: modules/stats.php [hunk 3]
```diff
@@ -188,30 +245,61 @@ function stats_get_options() {
...
+function stats_set_options( $options ) {
 	return update_option( 'stats_options', $options );
 }
 
+/**
+ * Stats Upgrade Options.
+ *
+ * @access public
+ * @param mixed $options Options.
+ * @return New Options.
```
Comment (eliorivero) [inline modules/stats.php]: Not ok, this should be an array.

File: modules/stats.php [hunk 4]
```diff
@@ -188,30 +245,61 @@ function stats_get_options() {
...
+	return stats_set_options( $options );
 }
 
-function stats_set_options($options) {
+/**
+ * Stats Set Options.
+ *
+ * @access public
+ * @param mixed $options Options.
+ * @return Update Options.
```
Comment (eliorivero) [inline modules/stats.php]: Ok, I'm not going to comment more about these. All these types must be fixed.

File: modules/stats.php [hunk 5]
```diff
@@ -488,44 +626,81 @@ function stats_reports_page( $main_chart_only = false ) {
...
+		$html );
 	return $html;
 }
 
+/**
+ * Stats Convert Post Title HTML
+ *
+ * @access public
+ * @param mixed $html HTML.
+ * @return void
```
Comment (eliorivero) [inline modules/stats.php]: This function does return something, a `string` and this doc says it returns nothing.

**Comment by zinigor** [comment]
@eliorivero thanks for pointing these out! We'll have to do a massive cleanout next time, but the point of this PR was to accept and merge certain changes from a user.

**Pull request merged**